clappr/clappr-ios
# Swift 4.2 migration

# Goal

To migrate Clappr to Swift 4.2

# How to test

1. Make sure that app runs and video plays
2. Run tests
